### PR TITLE
Feature/arm/bitcoind toml

### PR DESCRIPTION
**Pull request by arminsabouri**
The last commit is not my favorite. I wanted to read config from a toml file and prioritize that. Then fall back to cli args if needed. Lmk if there is a cleaner way

### CODE REVIEW

File: bin/btc-server/src/main.rs
Comment (BatiGencho) [inline bin/btc-server/src/main.rs]: I feel this could be put into its own separate method or even better From, TryFrom impl maybe ?

Comment (arminsabouri) [reply bin/btc-server/src/main.rs]: Good idea. A utility method may work best since it requies both cli config and file config

Comment (arminsabouri) [reply bin/btc-server/src/main.rs]: Moved to config.rs in [860767f](https://github.com/botanix-labs/Macbeth/pull/232/commits/860767f94cfa4f6a48396b312c41edf222a2237b)

**Review by BatiGencho — DISMISSED** [review]

**Review by scottmillner — DISMISSED** [review]

**Comment by arminsabouri** [comment]
Waiting for CI to pass. Will merge after

**Review by BatiGencho — APPROVED** [review]

**Pull request merged**
